### PR TITLE
Fix typo in condition for Windows Auth with LongPolling

### PR DESCRIPTION
**Pull request by BrennanConroy**
Just something I found while reading through code.

### CODE REVIEW

File: src/SignalR/common/Http.Connections/src/Internal/HttpConnectionDispatcher.cs [hunk 1]
```diff
@@ -504,7 +504,7 @@ private async Task<bool> EnsureConnectionStateAsync(HttpConnectionContext connec
                     // We specifically clone the identity on first poll if it's a windows identity
                     // If we swapped the new User here we'd have to dispose the old identities which could race with the application
                     // trying to access the identity.
-                    if (context.User.Identity is WindowsIdentity)
+                    if (context.User.Identity is WindowsIdentity == false)
```
Comment (anurse) [inline src/SignalR/common/Http.Connections/src/Internal/HttpConnectionDispatcher.cs]:
```suggestion
                    if (context.User.Identity isn't WindowsIdentity)
```

If only... :) (to be clear, this doesn't exist I'm just :trollface: ing)

Comment (mikaelm12) [reply src/SignalR/common/Http.Connections/src/Internal/HttpConnectionDispatcher.cs]: We should actually put parens around the first clause though.

Comment (BrennanConroy) [reply src/SignalR/common/Http.Connections/src/Internal/HttpConnectionDispatcher.cs]: Should we? There isn't a question of order of operations here, there is only one evaluation of the if statement.

Comment (mikaelm12) [reply src/SignalR/common/Http.Connections/src/Internal/HttpConnectionDispatcher.cs]: You don't *have* to, but I think it's a good idea to put parens around individual logical clauses when chaining. I wouldn't block the pr on it though.

Comment (halter73) [reply src/SignalR/common/Http.Connections/src/Internal/HttpConnectionDispatcher.cs]: I like the parens here. I think it just reads better. Or what about the following?

```suggestion
                    if (!(context.User.Identity is WindowsIdentity))
```

**Review by halter73 — APPROVED** [review]

File: src/SignalR/common/Http.Connections/src/Internal/HttpConnectionDispatcher.cs (hunk 1 above)
Comment (anurse) [reply src/SignalR/common/Http.Connections/src/Internal/HttpConnectionDispatcher.cs]: I like this better than `== false` I think. Someday we'll get the `is not` operator...

**Comment by BrennanConroy** [comment]
@aspnet/build Can someone merge please! The build hang isn't caused by this PR

**Pull request merged**